Two-Factor/README.md

```diff
@@ -8,51 +8,35 @@ Source: https://github.com/aerogear/aerogear-android-cookbook/tree/master/Two-Fa
 
 ## What is it?
 
-The ```AGDroid Two-Factor``` project demonstrates how to include OTP functionality in Android applications.
+_Two-Factor_ project demonstrates how to include OTP functionality in Android applications.
 
 ## How do I run it?
 
-### 0. System Requirements
-
-* [Java 7](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* [Gradle 2.2.1](http://www.gradle.org/)
-* Latest [Android SDK](https://developer.android.com/sdk/index.html) and [Platform version 22](http://developer.android.com/tools/revisions/platforms.html)
-
-### Configuring a testing server
-
-**Note**: The server side configuration is not mandatory for testing OTP with Android.
-
-### Configuring a testing server
+### System Requirements
 
-1. Follow directions to install [OTP-Demo](https://github.com/aerogear/aerogear-backend-cookbook/blob/master/OTP-demo/README.md)
-1. Open OTP backend app [http://localhost:8080/otp-demo](http://localhost:8080/otp-demo)
-1. Login with username: *user* and password: *password*.
-1. Now open this "OTP client application" on your phone
-1. Then scan the *Scan QRCode*
-1. Enter the current OTP on your mobile into the form
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Gradle](http://www.gradle.org/)
```

Comment:
In theory Gradle should download it automatically for you. I don't want to add the version again because we always forgot to update that kind of info.

@secondsun wdyt?